index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Disappear</title>
+
+<style>
+
+    .block {
+        display: flex;
+        justify-content: space-around;
+        padding: 100px;
+    }
+
+    .one-img {
+        width: 400px;
+        height: 300px;
+        opacity: 0;
+        transition: 1s;
+    }
+
+    .one-img:hover {
+        opacity: 100%
+    }
+
+    .two-img {
+        width: 0;
+        height: 0;
+        transition: 1s;
+        animation: appear 2s ease-in forwards;
+
+    }
+
+    @keyframes appear {
+        0% {
+            width: 0;
```

Comment:
Я не певен що варто робити анімацію `width` чи `height`. Вони здаєтья будуть перемальовувати лаяут?
Може краще наприклад:
```suggestion
@keyframes appear {
        0% {
          scale: 0;
        }
        100% {
          scale: 1;
        }
}
```
- https://web.dev/animations-guide/
- https://web.dev/animations-overview/
- https://web.dev/speed-unnecessary-paints/
- https://csstriggers.com/css-property-will-not-trigger-layout/